pkg/controller/gittrack/gittrack_controller.go

```diff
@@ -537,16 +537,19 @@ func (r *ReconcileGitTrack) Reconcile(request reconcile.Request) (reconcile.Resu
 	r.recorder.Eventf(instance, apiv1.EventTypeNormal, "CheckoutSuccessful", "Successfully checked out '%s' at '%s'", instance.Spec.Repository, instance.Spec.Reference)
 
 	// Attempt to parse k8s objects from files
-	objects, errors := objectsFrom(files)
-	if len(errors) > 0 {
-		sOpts.parseError = fmt.Errorf(strings.Join(errors, ",\n"))
+	objects, fileErrors := objectsFrom(files)
+	sOpts.ignoredFiles = fileErrors
+	sOpts.ignored += int64(len(fileErrors))
+
+	if len(fileErrors) > 0 {
 		sOpts.parseReason = gittrackutils.ErrorParsingFiles
 	} else {
 		sOpts.parseReason = gittrackutils.FileParseSuccess
 	}
 
 	// Update status with the number of objects discovered
-	sOpts.discovered = int64(len(objects))
+	sOpts.discovered = int64(len(objects)) + int64(len(fileErrors))
```

Comment:
I don't think we should include the number of errors in the `discovered` metric, as there could be multiple objects in an ignored file. Thoughts?

Reply:
Yeah, that's a good point, would be good however if we could still marshall the valid objects in a file even if there is one that is broken, that won't work right now I don't think 🤔